docs/resources/email_notification_integration.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Looks like this just wasn't caught from a previous change, I just made it when I ran make docs